docs/features/incremental-generators.cookbook.md

```diff
@@ -223,7 +223,7 @@ Provide that attribute in a `RegisterPostInitializationOutput` step. Register fo
 ```csharp
 public partial class UserClass
 {
-    [Generate]
+    [Generated]
```

Comment:
❓ Should we also qualify this? Either with `using GeneratedNamespace;` or as part of the attribute `[GeneratedNamespace.Generated]`?

Reply:
Yes, I think either would make it clearer.

Reply:
I applied @sharwell's suggestion to the PR